ueno/styles/ueno-black.json

```diff
@@ -0,0 +1,652 @@
+{
+	"settings": {
+		"border": {
```

Comment:
I think `appearanceTools` is missing, I wonder why this key keeps getting stripped from the export.

Reply:
Added to the main theme.json file in https://github.com/Automattic/themes/pull/6909/commits/b1b5aa2794892fc0772b006ce0b9bd4597085a3b